Documentation/network/clustermesh/clustermesh.rst

```diff
@@ -147,6 +147,11 @@ Example install using the Cilium CLI:
   cilium install --set cluster.name=$CLUSTER1 --set cluster.id=1 --context $CLUSTER1
   cilium install --set cluster.name=$CLUSTER2 --set cluster.id=2 --context $CLUSTER2
 
+.. note::
```

Comment:
This note in "Specify the Cluster Name and ID" section seems out of place. How about moving it to "Enable Cluster Mesh" section?

Reply:
I believe this comment has already been resolved by the original submitter:

https://github.com/cilium/cilium/pull/34174/files#diff-8c6c2ac85134f66ed70f9aca62524e18ac4b8003f7d41101521d023e491f3f82R199